src/main/java/org/omnifaces/util/Reflection.java

```diff
@@ -1011,4 +1011,74 @@ public static <T> T invokeStaticMethod(Class<?> type, String methodName, Object.
         }
     }
 
+    // Properties ---------------------------------------------------------------------------------------------------------
+
+    /**
+     * Internal utility to retrieve the index of the string after
+     * the usual prefix (get/is) of the accessor Method
+     * @return the index of the string array after the getter method prefix, -1 if not found
+     * @since 4.6
+     */
+    private static int propertyNameIndexOf(final char[] methodName) {
+        // todo: if we add an OR with methodName[0] == 's' when can support also setter methods
+        if ( methodName.length > 3 && methodName[0] == 'g' && methodName[1] == 'e' && methodName[2] == 't' ) {
+            return 3;
+        }
+        else if ( methodName.length > 2 && methodName[0] == 'i' && methodName[1] == 's' ) {
+            return 2;
+        }
+        else {
+            return -1;
+        }
+    }
+
+    /**
+     * @param method the accessor {@link Method}
+     * @return true if the passed {@link Method} is an accessor Method (it's name starts with "get" or "is")
+     * @since 4.6
```

Comment:
4.6 was already released. Next version is 4.7.

Reply:
right!